shared-data/protocol/types/schemaV7/command/setup.ts

```diff
@@ -72,12 +72,16 @@ export type SetupCreateCommand =
   | LoadLiquidCreateCommand
   | MoveLabwareCreateCommand
 
-export type LabwareLocation = { slotName: string } | { moduleId: string }
+export type LabwareLocation =
+  | 'offDeck'
+  | { slotName: string }
+  | { moduleId: string }
 
```

Comment:
to incorporate @jbleon95's labware adapter split work this a labware location might also hold a labware id right?
```suggestion
  | { moduleId: string }
  | { labwareId: string }
```

Reply:
I spoke with brian about this last week and he said to worry about that at a follow up time - but happy to add it here if @jbleon95's work is merged in?